felix/daemon/daemon.go

```diff
@@ -267,6 +267,16 @@ configRetry:
 			continue configRetry
 		}
 
+		ippoolKVList, err := backendClient.List(ctx, model.ResourceListOptions{Kind: apiv3.KindIPPool}, "")
+		if err != nil {
+			log.WithError(err).Error("Failed to list IP Pools")
+			time.Sleep(1 * time.Second)
+			continue configRetry
+		}
+		encapCalculator := calc.NewEncapsulationCalculator(configParams, ippoolKVList)
```

Comment:
isn't ippoolKVList going to be a model type list rather than an api/v3 type list?

Reply:
the values in the KVPairs are apiv3.IPPool, could I be doing something wrong when listing the pools (line 270)? is there a way to get a model.IPPool instead?